src/clap-saw-demo.cpp

```diff
@@ -744,7 +750,7 @@ void ClapSawDemo::handleNoteOff(int port_index, int channel, int n)
 {
     for (auto &v : voices)
     {
-        if (v.state != SawDemoVoice::OFF && v.key == n && v.portid == port_index &&
+        if (v.state != SawDemoVoice::OFF && v.state != SawDemoVoice::NEWLY_OFF && v.key == n && v.portid == port_index &&
```

Comment:
This one is a real edge case and it’s fine. When I do a sweep tonight I might add a little voice member is active to clean up all these if logics!